app/assets/stylesheets/views/_marketplace_landing_page.scss

```diff
@@ -272,7 +272,7 @@ $link-color: rgb(3,102,214);
   position: relative;
   display: flex;
   flex-direction: column;
-  background: transparent;
+  // background: transparent;
```

Comment:
remove dead code instead of leaving it commented, can always go back and add it in again